<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralwidget</class>
 <widget class="QWidget" name="centralwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>859</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="combobox_ports">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combobox_bauds">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_connection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>BAĞLAN</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>840</y>
     <width>1291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>20</width>
     <height>851</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>0</y>
     <width>20</width>
     <height>851</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_servoopen">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SERVO AÇMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_servoclose">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SERVO KAPAMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sendcommand">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>GÖNDER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_enginestop">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MOTOR STOP</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combobox_command">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UYDU STATÜSÜ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_enginerun">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MOTOR TAHRİK</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_run_value">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Manuel motor tahrik komutu için değer:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinbox_value">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button_refresh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_connect_tele">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>TELE</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_pl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_car">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>110</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableView" name="table_disabled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>1261</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_pr_hei">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_sp_tmp_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>380</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_pl_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_pl_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_pl_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
